src/rpc/blockchain.cpp

```diff
@@ -2219,7 +2223,12 @@ static RPCHelpMan savemempool()
         throw JSONRPCError(RPC_MISC_ERROR, "Unable to dump mempool to disk");
     }
 
-    return NullUniValue;
+    auto node_context = util::AnyPtr<NodeContext>(request.context);
```

Comment:
Maybe just move this to the top with EnsureAnyMemPool...

Reply:
Done in 0f6f763